client/src/views/Simulation/components/RunButton.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="btn-group">
+    <button
+      type="button"
+      class="btn btn-primary"
+      :class="{ 'blue' : !autoRunInput }"
+      @click="onClickRun"
+    >
+      <font-awesome-icon :icon="['fas', ( autoRunInput ? 'pause' : 'play') ]" />
+      {{ displayText }}
+    </button>
+    <button
+      type="button"
+      class="btn btn-primary blue dropdown-toggle dropdown-toggle-split"
+      @click="dropdownOpen = !dropdownOpen"
+    />
+    <div class="dropdown-menu dropdown-menu-right" :class="{ 'show': dropdownOpen }">
+      <input type="checkbox" v-model="autoRunInput" id="auto-run-input" />
+      <label
+        for="auto-run-input"
+        title="Enable model to run automatically on when Parameters are updated">
+        Auto-Run
+      </label>
+      <div class="dropdown-divider" />
+      <h6 class="dropdown-header">_Sequence_ Settings</h6>
+      <div class="run-config">
+        <label for="run-config-start">Start</label>
+        <input type="number" id="run-config-start" v-model.number="configInput.start" />
+        <label for="run-config-end">End</label>
```

Comment:
It doesn't seem changing a value here is getting reflected in the variables time-series charts. How is this getting passed to the variables panel?

Reply:
You need to re-run the simulation are those are settings on the run button.
It will be easy to do the re-run on value changes.